src/cli/install.js

```diff
@@ -0,0 +1,90 @@
+const CONFIG = require('../constants/config')
```

Comment:
This file could have been done in bash, but I feel like JS is easier and more understandable... probably because I'm a JS dev. 🙃

Reply:
Array.join is just for readability.